app.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics-controllers");
+
+const app = express();
+
+app.use = (express.json())
```

Comment:
Please do not include code that is not currently required by the program. You have no functionality that needs this. Yet!

Unnecessary code can be confusing for other devs (and you!) and as such is bad practise.